test/test_operations.py

```diff
@@ -469,6 +469,21 @@ def runAtenTest(self, tensors, fn, device=None, rel_err=1e-2, abs_err=1e-5):
           rel_err=rel_err,
           abs_err=abs_err)
 
+  def runGradTest(self, tensors, fn, device=None, rel_err=1e-2, abs_err=1e-5):
```

Comment:
This is an almost exact copy of the one above.
I'd make a generic one which takes a lambda on how to create XLA tensors, and call it from the two places.

Reply:
In fact, runAtenTest can be change to work with both cases.